module/VuFind/src/VuFind/View/Helper/Bootstrap3/Search.php

```diff
@@ -58,7 +58,8 @@ protected function getContainerClass()
      */
     protected function renderExpandLink($url, $view)
     {
-        return '<a href="' . $url . '" title="' . $view->transEsc('spell_expand_alt')
-            . '"><i class="icon-circle-arrow-right"></i></a>';
+        return ' <a href="' . $url
```

Comment:
@crhallberg, I'm confused about what is going on here; you have changed renderExpandLink() here, but you have also eliminated the code in AbstractSearch that calls renderExpandLink(), ensuring that this version of the function will never be called. I think the correct solution is to make sure that the code is correctly fixed here and leave AbstractSearch as it is... but if I'm missing something, please let me know.